### ISSUE TITLE
[Issue]: Fold6 not detecting as game

### ISSUE THREAD

**Issue by nik-joseph** [ClassicOldSong/moonlight-android#54]
### Describe the bug

Installed the game release version on samsung Fold 6 and the app isnt being detected as game and game booster isnt triggered.

### Steps to reproduce

Install game release apk on Fold 6

### Other Moonlight clients

PC

### Moonlight adjusted settings

None

### Moonlight default settings

Yes

### Gamepad-related connection issue

No

### Gamepad-related input issue

No

### Gamepad-related streaming issue

No

### Android version

Android 14

### Device model

Samasung Galaxy Fold 6

### Server PC OS version

Windows 11 Pro 23H2

**Comment by ClassicOldSong** [comment]
Does the app‘s name shown as `Artemis (Game)` ?

**Comment by nik-joseph** [comment]
> Does the app‘s name shown as `Artemis (Game)` ?

Yup. I tried adding it to Gaming Hub and it says its not a game as well.

**Comment by nik-joseph** [comment]
This is what is says on adding to Gaming Hub (cropped screenshot)
![screenshot](https://github.com/user-attachments/assets/6cd99861-6557-4d3c-bf51-5431df0bf4d9)

**Comment by ClassicOldSong** [comment]
😢 Vendors all have their own different ways to detect games... I don't have a Samsung device but it shows correctly as game on all my other phones... And some people say Samsung removed Gaming Hub from its store so they can't configure games anymore...

IDK what's happening but the GAME virant seems working for them.

Ref: https://github.com/ClassicOldSong/moonlight-android/issues/48

**Comment by nik-joseph** [comment]
Oh, guess will have to wait till someone else figures out what's wrong with what I did.
I haven't either bothered with gaming hub tbh, I checked it out after the overlay didn't show.

If it helps the playstore moonlight app has the overlay working.

**Comment by LuisB79** [comment]
> 😢 Vendors all have their own different ways to detect games... I don't have a Samsung device but it shows correctly as game on all my other phones... And some people say Samsung removed Gaming Hub from its store so they can't configure games anymore...
> 
> IDK what's happening but the GAME virant seems working for them.
> 
> Ref: #48

actually no, i hadn't had the time to check, it is not being detected as a game.

**Comment by LuisB79** [comment]
> Oh, guess will have to wait till someone else figures out what's wrong with what I did. I haven't either bothered with gaming hub tbh, I checked it out after the overlay didn't show.
> 
> If it helps the playstore moonlight app has the overlay working.

you didn't do anything wrong afaik

**Comment by ClassicOldSong** [comment]
> > 😢 Vendors all have their own different ways to detect games... I don't have a Samsung device but it shows correctly as game on all my other phones... And some people say Samsung removed Gaming Hub from its store so they can't configure games anymore...
> 
> > 
> 
> > IDK what's happening but the GAME virant seems working for them.
> 
> > 
> 
> > Ref: #48
> 
> 
> 
> actually no, i hadn't had the time to check, it is not being detected as a game.

Hmmmmm

I'll check that later but they're being detected as game on non Samsung phones 🫠

What a mess of Android fragmentation...

**Comment by ClassicOldSong** [comment]
https://gist.github.com/Adnan-Bacic/718eb3b4e70380696c91dc21c4804112

Just found this

Have to mail to Samsung about it.... They white list apps for being games, but not allowing users to mark apps manually.......

Guess I don't need to release the game variant in the future if the main one can be listed as game.

**Comment by TerminatedGB** [comment]
> https://gist.github.com/Adnan-Bacic/718eb3b4e70380696c91dc21c4804112
> 
> Just found this
> 
> Have to mail to Samsung about it.... They white list apps for being games, but not allowing users to mark apps manually.......
> 
> Guess I don't need to release the game variant in the future if the main one can be listed as game.

I own a Samsung S24 Ultra and also experienced the same issue, I think you are right about the problem

**Comment by Pradinhodot** [comment]
> > https://gist.github.com/Adnan-Bacic/718eb3b4e70380696c91dc21c4804112
> > 
> > Just found this
> > 
> > Have to mail to Samsung about it.... They white list apps for being games, but not allowing users to mark apps manually.......
> > 
> > Guess I don't need to release the game variant in the future if the main one can be listed as game.
> 
> I own a Samsung S24 Ultra and also experienced the same issue, I think you are right about the problem

I also have an S24 Ultra and the same problem, it doesn't appear as a game in game booster, the performance could be much better if it weren't for that.
Do we have any news about this yet?

**Comment by Pradinhodot** [comment]
> > https://gist.github.com/Adnan-Bacic/718eb3b4e70380696c91dc21c4804112
> > 
> > Just found this
> > 
> > Have to mail to Samsung about it.... They white list apps for being games, but not allowing users to mark apps manually.......
> > 
> > Guess I don't need to release the game variant in the future if the main one can be listed as game.
> 
> I own a Samsung S24 Ultra and also experienced the same issue, I think you are right about the problem

I also have an S24 Ultra and the same problem, it doesn't appear as a game in game booster, the performance could be much better if it weren't for that.
Do we have any news about this yet?

**Comment by ClassicOldSong** [comment]
Just submitted a support ticket to Samsung, waiting for their replies.

**Comment by abdulsametoger** [comment]
I have this problem too but solved. Use ApkTool M or apk editor app and change package name to "com.limelight" and install this. Probably, gaming hub has whitelist or whatever that if app is verified game, system detect it and automatically add to gaming hub. 

![Image](https://github.com/user-attachments/assets/0401eb90-ed18-4a1c-8e81-9edd8f76ddfe)
![Image](https://github.com/user-attachments/assets/1f3d87b3-fc04-43ef-9144-3b264eb9537b)

**Comment by Pradinhodot** [comment]
> I have this problem too but solved. Use ApkTool M or apk editor app and change package name to "com.limelight" and install this. Probably, gaming hub has whitelist or whatever that if app is verified game, system detect it and automatically add to gaming hub. 
> 
> ![Image](https://github.com/user-attachments/assets/0401eb90-ed18-4a1c-8e81-9edd8f76ddfe)
> ![Image](https://github.com/user-attachments/assets/1f3d87b3-fc04-43ef-9144-3b264eb9537b)

Can you please explain better? I'll try this for sure

**Comment by abdulsametoger** [comment]
-Download latest Artemis apk
-Use "Apktool M" app 
-in Apktool M app, select downloaded Artemis apk and edit the apk before install
-Artemis app package name is "com.limelight.noir" 
-Edit package name to "com.limelight" then save
-Install edited apk
-Finish

**Comment by Pradinhodot** [comment]
> -Download latest Artemis apk
> -Use "Apktool M" app 
> -in Apktool M app, select downloaded Artemis apk and edit the apk before install
> -Artemis app package name is "com.limelight.noir" 
> -Edit package name to "com.limelight" then save
> -Install edited apk
> -Finish
> 
> 

Teste this, It really Works, but i guess he cant do this because its the APK name of the original moonlight, although i need to test If the stutters are really gone

**Comment by furiseto** [comment]
My Fold4's Game Booster has now detected Artemis as game.

**Comment by nik-joseph** [comment]
Can confirm latest apk being detected as a game in Fold6

Edit: apk version: v12.1.250118

**Comment by shanebeeps** [comment]
To follow up on this, my Samsung Galaxy S25 Ultra does NOT see the latest "nonRoot_game-release" as a game. It installs as "Artemis" and when I attempt to add it to Gaming Hub it tells me it is not a game. I have not attempted to use an APK name adjustment yet as I figured you would want to know current behavior.

**Comment by ClassicOldSong** [comment]
Samsung has added Artemis to their game list and multiple users have confirmed it working. You need to remove and reinstall Artemis.

**Comment by shanebeeps** [comment]
> Samsung has added Artemis to their game list and multiple users have confirmed it working. You need to remove and reinstall Artemis.

I've removed it from Gaming Hub, uninstalled Artemis, reinstalled Artemis, attempted to re-add to GH and same error. Do I need to uninstall Moonlight proper or something odd like that?

**Comment by ClassicOldSong** [comment]
Well I'm not Samsung so I don't know how they work exactly.

Someone said adding to Gamehub will make the app defaults to 1080P so that might not be ideal somehow.

Does S25 still have vrr issues? If not then Gamehub isn't necessary I think.

**Comment by DyingHeartD** [comment]
There's a new Samsung app called "Game booster +" that lets you force any app to be detected as a game.

[quickshare.samsungcloud.com/pFXCyMajmA2V](url)

![Image](https://github.com/user-attachments/assets/db674796-a319-47a0-9af0-f9c1388db9c8)
